modules/base/admin/objects.py

```diff
@@ -192,14 +193,28 @@ def git_clone(path: Path, url: str) -> Optional[str]:
             stderr: str = str(exc)[str(exc).find("stderr: ") + 8 :]
             return stderr
 
-    def git_pull(self) -> str:
+    def git_pull(self, force: bool = False) -> str:
         """Perform 'git pull' over the repository.
 
         :return: Git output
         """
         is_base: bool = self.name == "base"
         repo = git.repo.base.Repo(str(self.path), search_parent_directories=is_base)
-        result: str = repo.git.pull(force=True)
+        result: str = repo.git.pull(force=force)
+        return result
+
+    def git_reset(self) -> str:
```

Comment:
Since it does reset & pull, should this be named `git_reset_pull()` instead?

Reply:
You're right, changed.